lms/templates/problem.html

```diff
@@ -56,6 +56,11 @@ <h2 class="problem-header">
     % if check_button:
     <input class="check ${ check_button }" type="button" data-checking="${ check_button_checking }" value="${ check_button }" />
     % endif
+    % if exceeded_time_limit:
+    <p class="exceeded-time-limit-message">
```

Comment:
What about just making this a disabled button and just take the styling that gives you? 
i.e.: ```<input class="check disabled" type="button" value="Deadline has passed, you can no longer submit." />```
@jinpa & @stvstnfrd ?